go/analysis/passes/printf/printf.go

```diff
@@ -66,6 +68,49 @@ argument list. Otherwise it is assumed to be Print-like, taking a list
 of arguments with no format string.
 `
 
+// Kind is a kind of fmt function behavior
+type Kind int
+
+const (
+	KindNone   Kind = iota // not a fmt wrapper function
+	KindPrint              // function behaves like fmt.Print
+	KindPrintf             // function behaves like fmt.Printf
+)
+
+func (kind Kind) String() string {
+	switch kind {
+	case KindPrint:
+		return "print"
+	case KindPrintf:
+		return "printf"
+	}
+	return ""
+}
+
+// Result is the printf analyzer's result type. Clients may query the
+// result to learn whether a function is a print wrapper.
```

Comment:
Let's say "whether a function behaves like fmt.Print or Printf" in the documentation.